python/pacsign/PACSign.md

```diff
@@ -1,11 +1,11 @@
 # PACSign #
 
 ## SYNOPSIS ##
-`python PACSign.py [-h] {FIM,SR,BBS,BMC,BMC_FW,AFU,PR,GBS} ...`
+`python PACSign.py [-h] {FIM,SR,BBS,FIM2,SR2,BBS2,BMC,BMC_FW,AFU,PR,GBS} ...`
 
 `python PACSign.py <CMD>  [-h] -t {UPDATE,CANCEL,RK_256,RK_384} -H HSM_MANAGER
                           [-C HSM_CONFIG] [-r ROOT_KEY] [-k CODE_SIGNING_KEY]
-                          [-d CSK_ID] [-i INPUT_FILE] [-o OUTPUT_FILE] [-y] [-v]`
+                          [-d CSK_ID] [-S] [-i INPUT_FILE] [-o OUTPUT_FILE] [-y] [-v]`
```

Comment:
wouldn't -s be more consistent than -S?

Reply:
I consider this an infrequently used option, soon to be deprecated as we're not using 256 any more, so I thought the capital would be best.  It's slightly more effort, and we may want to use lower-case for something else in the future.  I can obviously change it if you think it's important...